sample-apps/python-manual-instrumentation-sample-app/app.py

```diff
@@ -78,29 +80,30 @@ def setup_opentelemetry(tracer, meter):
     tracer = trace.get_tracer(__name__)
 
     # Setting up Metrics
-    metric_reader = PeriodicExportingMetricReader(OTLPMetricExporter())
+    metric_reader = PeriodicExportingMetricReader(exporter=OTLPMetricExporter(), export_interval_millis= 1000)
```

Comment:
nit: consistency export_interval_millis= 1000 -> `export_interval_millis=1000`

Reply:
Changed in my latest commit